site/templates/collection/wishlist.jet

```diff
@@ -24,11 +24,11 @@
 
 {{end}}
   <section class="page-collection" aria-label="{{i18n("wcag_aria_label_wishlist")}}">
-    <div class="swiper-wrapper-container">
+    <div class="collection-wrapper-container">
       {{if config("default_image_type") == "portrait"}}
-        <div class="swiper-container" is-wishlist="true" data-items-per-row="6" data-layout="portrait">
+        <div class="collection-container" swiper-ignore="true" data-items-per-row="6" data-layout="portrait">
       {{else}}
-        <div class="swiper-container" is-wishlist="true" data-items-per-row="4" data-layout="landscape">
+        <div class="collection-container" swiper-ignore="true" data-items-per-row="4" data-layout="landscape">
```

Comment:
Are you sure that `is-wishlist` isn't used by anything else?

Reply:
Reverted to is-wishlist (due to reverting js changes)